src/commands/deploy/metadata.ts

```diff
@@ -156,6 +164,10 @@ export default class DeployMetadata extends SfCommand<DeployResultJson> {
       if (flags['dry-run']) this.log('Dry-run complete.');
     }
 
+    if (result.response.status === RequestStatus.Succeeded) {
```

Comment:
I can't make a comment about line 157 because it's not changing in this PR, but I think we need to change it somewhere.  

SDR is trying to do some smarter things with the polling freq based on size of the deployment.  You probably want to omit the frequency param to take advantage of those.
 
https://github.com/forcedotcom/source-deploy-retrieve/blob/e97744f243d943c85ca8df22e004c278fbad0715/src/client/metadataTransfer.ts#L227